app/assets/stylesheets/opportunities.scss

```diff
@@ -28,6 +32,10 @@ h1, h2 {
   color: white;
   text-align: center;
   margin-bottom: 5%;
+  &:hover {
+    background-color: #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`